charts/prometheus-pushgateway/templates/secret.yaml

```diff
@@ -8,3 +8,14 @@ metadata:
 data:
   web-config.yaml: {{ include "prometheus-pushgateway.webConfiguration" . | b64enc}}
 {{- end }}
+---
+apiVersion: v1
+kind: Secret
+metadata:
```

Comment:
Since the secret is meant for a ServiceMonitor, the target namespace has to follow that of the ServiceMonitor (see `serviceMonitor.namespace`).